### PR TITLE
Update nickname rules

### PR DESCRIPTION
**Pull request by Akarys42**
Implements suggestion 1052159108557570088 ("Add a rule in the nicknames section of the rules against intentionally confusing or misleading nicknames, to avoid accessibility issues and impersonations.") and simplify the wording of that section.

### CODE REVIEW

**Comment by Akarys42** [comment]
[Rendered view](https://update-nickname-rules.quiltmc-org.pages.dev/en/community/rules/#nicknames)

**Comment by obj-obj** [comment]
> no Zalgo

nooooooooooooooo

**Comment by Southpaw1496** [comment]
I have a few thoughts about the wording. I'm not sure how much of this is "French grammar is different than English" and how much is "Southpaw enforcing his writing style on everyone else" but I thought I'd put them out there anyway.

1. Maybe I'm just not hip or down with the language kids, but I'd never use `ex:` to abbreviate "for example", I'd use `eg.` instead.
2. I'm not sure if you're using "example" correctly in the first place. I personally think it would be easier to read if you got rid of the example part and just put them straight.

Here are some examples. I chopped and changed them a little, feel free to take selectively.

> 1.  Nicknames must be easy to read, especially by a screen-reader. This means no Zalgo or other difficult-to-read styling, or Unicode control characters.
>  2. Nicknames cannot be intentionally confusing or misleading. Impersonation is stricly forbidden, as well as using similar looking nicknames to be intentionally confusing.
>  3. Nicknames must have some readable characters. This means no "empty" nicknames.
4. Are empty nicknames the only thing Rule 3 is trying to prevent? If so, couldn't we just say "no empty nicknames", and if not, perhaps we should give another example?
5. I'm not sure if the last paragraph needs to change at all, I've always thought our right to change nicknames that break rules was implicit. If you think it does, you should remove "also".
6. The first paragraph could be much shorter. "All nicknames must adhere to the rules above, in addition to the following rules:"

I think it looks good overall, though.

**Comment by Akarys42** [comment]
The examples are because they are only one way the rule would apply. For example, not having a hard to read nickname means you can't have zalgo, but also anything else hard to read that isn't zalgo. Does that make sense?

Appreciate the feedback, I'll have a look at it shortly.

**Comment by Southpaw1496** [comment]
I'm going to botch saying this, but while I understand what you're saying about the examples, I don't think my versions imply that only the specific examples we give are banned, but if you did want to explicitly say "example", you shouldn't describe a good nickname "easy to read", before giving examples of bad ones. So, instead, you might say:

> 1. Nicknames can't contain characters which might be difficult to read, especially for screen readers (eg.  Zalgo, ornate  Unicode letters, or Unicode control characters.)
> 2. Nicknames can't be intentionally confusing or misleading (eg. nicknames that impersonate someone else, or are intentionally confusingly similar to someone else's.
> 3. No nicknames without readable characters (e.g "invisible" names).

**Comment by BucketOfCompasses** [comment]
Unicode defines zero width joiners as control characters so rule 1 would forbid certain system tags, can we make an exception for these...

**Comment by gdude2002** [comment]
Out of curiosity, how are those useful in system tags?

**Comment by Akarys42** [comment]
Discord will strip ZWS from names anyway (including webhooks), there is no need to complexity the rules for that

File: src/pages/en/community/rules.mdx [hunk 1]
```diff
@@ -86,9 +86,12 @@ Do not change your avatar, nickname, username or custom/rich status to try to ev
 
 #### Nicknames
 
-All nicknames must adhere to the rules above. Additionally, they should not be hard to read or too noisy. Nicknames may be considered difficult to read or noisy if they contain emojis, Zalgo or other difficult-to-read styling, or Unicode control characters. "Empty" nicknames are also considered difficult to read.
+All nicknames must adhere to the rules above. Additionally, nicknames have to adhere to the following rules:
+1. Nicknames must be easy to read, especially by a screen-reader (ex: no Zalgo or otherwise hard to read non-ascii characters, with the exception of non-Latin names)
+2. Nicknames cannot be intentionally confusing or misleading (ex: impersonation is stricly forbidden, as well as using similar looking nicknames to be intentionally confusing)
```
Comment (pluiedev) [inline src/pages/en/community/rules.mdx:91]: typo: `stricly` -> `strictly`
Also I'd probably phrase this differently without `(ex: )`s:

```
2. Nicknames cannot be intentionally confusing or misleading — impersonation is strictly forbidden, as well as using similar-looking nicknames to intentionally confuse others.

File: src/pages/en/community/rules.mdx [hunk 2]
```diff
@@ -86,9 +86,12 @@ Do not change your avatar, nickname, username or custom/rich status to try to ev
 
 #### Nicknames
 
-All nicknames must adhere to the rules above. Additionally, they should not be hard to read or too noisy. Nicknames may be considered difficult to read or noisy if they contain emojis, Zalgo or other difficult-to-read styling, or Unicode control characters. "Empty" nicknames are also considered difficult to read.
+All nicknames must adhere to the rules above. Additionally, nicknames have to adhere to the following rules:
+1. Nicknames must be easy to read, especially by a screen-reader (ex: no Zalgo or otherwise hard to read non-ascii characters, with the exception of non-Latin names)
```
Comment (pluiedev) [inline src/pages/en/community/rules.mdx:90]: I'd probably emphasize that the screen reader part is just an example of making your nickname accessible. Also, should we define what "hard-to-read" means? If we shouldn't then we should clarify that nicknames are reviewed on a case-by-case basis by moderators and are subject to the moderators' subjective scrutiny.

File: src/pages/en/community/rules.mdx [hunk 3]
```diff
@@ -86,9 +86,12 @@ Do not change your avatar, nickname, username or custom/rich status to try to ev
...
+2. Nicknames cannot be intentionally confusing or misleading (ex: impersonation is stricly forbidden, as well as using similar looking nicknames to be intentionally confusing)
+3. Nicknames must have some readable characters (ex: no "empty" nicknames)
 
-We also reserve the right to change or reset nicknames that are designed to place you higher in the member list in community spaces that have one, which is known as "hoisting".
+We also reserve the right to change or reset nicknames that are infringing those rules, or designed to place you higher in the member list in community spaces that have one, which is known as "hoisting".
```
Comment (pluiedev) [inline src/pages/en/community/rules.mdx:94]: Maybe we should just put no hoisting in the rules to make the rule set more consistent

**Comment by Akarys42** [comment]
> Also, should we define what "hard-to-read" means?
I don't think so, that'd open to rule lawyering quite easily. That should
be left to the moderator's appreciation imo.

> Maybe we should just put no hoisting in the rules to make the rule set
more consistent
If we do that, there will be a set expectation that we'll be taking down
any hoister, which honestly seems like a waste of time to me. If anything,
I'd rather completely stop caring about hoisters, but some people like to
reset their nicks.

**Comment by pluiedev** [comment]
>If anything I'd rather completely stop caring about hoisters, but some people like to reset their nicks.
Yeah honestly I agree, it's not like hoisters actually cause real problems for us anyway...

**Comment by woodiertexas** [comment]
Afaik hoisting just makes the memberlist look ugly.

**Comment by pluiedev** [comment]
> Afaik hoisting just makes the memberlist look ugly.

Ugly? Possibly. Distracting? Probably not? We already have lots of staff members and bots hoisted on there that normal hoisters don't really show up that much. A moderation problem either way? Nope

**Comment by woodiertexas** [comment]
Only way it would be a moderation problem is if people combine hoisting with status advertisements,

**Review by Southpaw1496 — APPROVED** [review]
